spec/index.md

```diff
@@ -4942,7 +5003,7 @@ if A.settings.reserved_cycles_limit is not null:
 else:
   New_reserved_balance_limit = 5_000_000_000_000
 
-Cycles_reserved = cycles_to_reserve(S, Canister_id, New_compute_allocation, New_memory_allocation, EmptyCanister.wasm_state)
+Cycles_reserved = cycles_to_reserve(S, Canister_id, New_compute_allocation, New_memory_allocation,  S.snapshots[A.canister_id], EmptyCanister.wasm_state)
```

Comment:
the canister has just been created so there's no snapshot yet, we should use `null` here and also define `snapshots[Canister_id] = null` in the "State after" section.

Reply:
Ok, I was thinking that this could be interpreted as an empty list. But I will use null to make it clearer.